discovery-provider/alembic/versions/42d5afb85d42_create_audio_transactions_history.py

```diff
@@ -0,0 +1,68 @@
+"""create audio_transactions_history
+
+Revision ID: 42d5afb85d42
+Revises: a8752810936c
+Create Date: 2022-10-03 18:29:24.661763
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from src.utils.alembic_helpers import table_exists
+
+# revision identifiers, used by Alembic.
+revision = "42d5afb85d42"
+down_revision = "a8752810936c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # audio_transactions_history
+    if not table_exists("audio_transactions_history"):
+        op.create_table(
+            "audio_transactions_history",
+            sa.Column("user_bank", sa.String(), nullable=False),
+            sa.Column("signature", sa.String(), nullable=False),
+            sa.Column("transaction_type", sa.String(), nullable=False),
+            sa.Column("method", sa.String(), nullable=False),
+            sa.Column(
+                "created_at",
+                sa.DateTime(),
+                nullable=False,
+                server_default=sa.func.current_timestamp(),
+            ),
+            sa.Column(
+                "updated_at",
+                sa.DateTime(),
+                nullable=False,
+                server_default=sa.func.current_timestamp(),
+            ),
+            sa.Column("change", sa.BigInteger(), nullable=False),
+            sa.Column("balance", sa.BigInteger(), nullable=False),
+            sa.Column("tx_metadata", sa.String(), nullable=False),
+            sa.PrimaryKeyConstraint("signature", "user_bank"),
```

Comment:
Maybe should be user_bank first? to make getting all the transactions by a user faster?